marimo/_cli/cli.py

```diff
@@ -533,6 +535,15 @@ def new(
     type=bool,
     help="Include notebook code in the app.",
 )
+@click.option(
+    "--ttl",
```

Comment:
Do you think there will be other ttls? If so, would session-ttl be better?

Reply:
Yea I like `session-ttl` better, thanks